docs/docs/getting-started/install.md

```diff
@@ -92,6 +92,31 @@ curl -LO https://github.com/edgelesssys/constellation/releases/latest/download/c
 sudo install constellation-darwin-amd64 /usr/local/bin/constellation
 ```
 
+</tabItem>
+
+<tabItem value="windows-amd64" label="Windows (amd64)">
+
+1. Download the CLI:
+
+```bash
+Invoke-WebRequest -OutFile ./constellation.exe -Uri 'https://github.com/edgelesssys/constellation/releases/latest/download/constellation-windows-amd64.exe'
+```
+
+2. [Verify the signature](../workflows/verify-cli.md) (optional)
+
+3. Install the CLI under `C:\Program Files\Constellation\bin\constellation.exe`
+
+3. Add the CLI to your PATH:
+
+    1. Open `Advanced system settings` by searching for the App in the Windows search
+    2. Go to the `Advanced` tab
+    3. Click `Environment Variables…`
+    4. Click variable called `Path` and click `Edit…`
+    5. Click `New`
+    6. Enter the path to the folder containing the binary you want on your PATH: `C:\Program Files\Constellation\bin`
+
+4. In all documentation, if `constellation` doesn't work as the binary name, try `constellation.exe` or the full path instead: `C:\Program Files\Constellation\bin\constellation.exe`
```

Comment:
```suggestion
```
I would just omit this section, since doing the above should make `constellation` globally available for the user